stan/math/prim/mat/prob/bernoulli_logit_glm_lpmf.hpp

```diff
@@ -107,7 +110,7 @@ typename return_type<T_x, T_alpha, T_beta>::type bernoulli_logit_glm_lpmf(
   T_partials_return theta_derivative_sum = 0;
   T_partials_return exp_m_ythetan;
   for (size_t n = 0; n < N; ++n) {
-    ytheta[n] += signs[n] * value_of(alpha_vec[n]);
+    check_finite(function, "Matrix of independent variables", ytheta[n]);
     exp_m_ythetan = exp(-ytheta[n]);
     if (ytheta[n] > cutoff) {
       logp -= exp_m_ythetan;
```

Comment:
I'm looking at line 122. Can't we take this out of the loop and just sum it there?